skqulacs/circuit/pre_defined.py

```diff
@@ -518,30 +518,16 @@ def pooling_circuit(
         circuit.add_CNOT_gate(this_bit, next_bit)
         circuit.add_Z_gate(next_bit)
 
-    # # depth 1 (0, 1)
-    # circuit = conv_circuit(circuit, 0, 1)
-    # circuit = pooling_circuit(circuit, 0, 1)
-    # circuit = conv_circuit(circuit, 2, 3)
-    # circuit = pooling_circuit(circuit, 2, 3)
-    # circuit = conv_circuit(circuit, 4, 5)
-    # circuit = pooling_circuit(circuit, 4, 5)
-    # circuit = conv_circuit(circuit, 6, 7)
-    # circuit = pooling_circuit(circuit, 6, 7)
-
-    # # depth 2 (1, 3)
-    # circuit = conv_circuit(circuit, 1, 3)
-    # circuit = pooling_circuit(circuit, 1, 3)
-    # circuit = conv_circuit(circuit, 5, 7)
-    # circuit = pooling_circuit(circuit, 5, 7)
-
-    # # depth 3 (3, 7)
-    # circuit = conv_circuit(circuit, 3, 7)
-    # circuit = pooling_circuit(circuit, 3, 7)
-
-    # 二分木を作成してペアを作る
-    # 枝から幅優先で二分木を作らないと精度が悪くなる。。
     targets = []
 
+    # 0始まりの数字のリストを受け取り
+    # 二分木でペアを作ります
+    # [0,1,2,3,4,5,6,7]を指定した場合
+    # [[0, 1], [2, 3], [1, 3], [4, 5], [6, 7], [5, 7], [3, 7]] になります。
+    # [0, 1],[2, 3],[4, 5],[6, 7]が枝となり、
+    # 次の階層の[1, 3],[5, 7]となります。階層の数字は下の層の番号が大きい方通しがペアになります。
```

Comment:
意味が通らなさそうだったんですが、こうですかね?
```suggestion
    # 次の階層の[1, 3],[5, 7]となります。階層の数字は下の層の通し番号が大きい方がペアになります。
```